p2p/antiflood/factory/disabledAntiFlood.go

```diff
@@ -0,0 +1,31 @@
+package factory
+
+import (
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type disabledAntiFlood struct {
+}
+
+// ResetForTopic won't do anything
+func (n *disabledAntiFlood) ResetForTopic(topic string) {
+}
+
+// SetMaxMessagesForTopic won't do anything
+func (n *disabledAntiFlood) SetMaxMessagesForTopic(topic string, maxNum uint32) {
```

Comment:
unused variables should be replaced with _

Reply:
+1

Reply:
done